Sources/SIL/SILPrinter.swift

```diff
@@ -163,6 +170,15 @@ class SILPrinter: Printer {
         case let .metatype(type):
             print("metatype ")
             print(type)
+        case let .partialApply(calleeGuaranteed, onStack, value, substitutions, arguments, type):
+            print("partial_apply ")
+            print(when: calleeGuaranteed, "[callee_guaranteed] ")
+            print(when: onStack, "[on_stack] ")
+            print(value)
+            print(whenEmpty: false, "<", substitutions, ", ", ">") { naked($0) }
+            print("(", arguments, ", ", ")") { print($0) }
+            print(" : ")
+            print(type)
         case let .pointerToAddress(operand, strict, type):
             print("pointer_to_address ")
             print(operand)
```

Comment:
Am I missing something or this PR doesn't have instruction tests?

Reply:
Oh I must have forgotten to add those. Will do.